src/doc/gui_manual/Preferences/File_Locations.rst

```diff
@@ -0,0 +1,271 @@
+.. _file_locations:
+
+Configuration File Locations
+----------------------------
+
+VisIt_ reads and writes information about aspects of its *configuration*
+to various files. In most cases where VisIt_ provides controls to either save
+or load some aspect of its configuration, the user is presented with a file
+browser dialog and can explicitly choose arbitrary locations on the file system
+to look for or store files. However, this is not universally true and VisIt_
+does have *prescribed* locations for some configuration files. In this section
+we provide some additional details about these various files.
+
+The locations of prescribed files may depend on the platform upon which VisIt_
+is running. Typically, on UNIX and OSX systems, prescribed configuration files
+are stored in ``~/.visit`` whereas on Windows systems, they are, by default, in
+``%USERPROFILE%\Documents\VisIt``, which may be something like
+``C:\Users\<user-name>\Documents\VisIt``. Furthermore, on Windows, Visit_ honors
+the ``CSIDL_PERSONAL`` and ``CSIDL_MYDOCUMENTS``
+`CSIDL environment variables <https://docs.microsoft.com/en-us/windows/win32/shell/csidl>`_.
+Depending on the how the system is configured, these might actually resolve to a
+networked drive, but most commonly, to the values described previously. Finally,
+Windows users can also set the ``VISITUSERHOME`` environment variable to point
+to whatever location they desire, and VisIt_ will use that location instead.
+In the descriptions below, we use the symbol ``VUSER_HOME`` as a way to refer to
+whatever this location happens to be on whatever platform the user is running.
+
+In addition, when running VisIt in :ref:`client/server mode <Client-Server Mode>`,
+the user may need to be aware of the location of these files on either the client
+or the server or both.
+
+Files in ``VUSER_HOME``
+~~~~~~~~~~~~~~~~~~~~~~~
+Most of the files associated with VisIt_ configuration have their prescribed home
+in ``VUSER_HOME``. When running in client/server, it is the configuration files
+on the *local client* that effect behavior. This means they are always the
+files on the *local* machine and not the *remote* system. Any configuration
+files that might also be on the remote server do not play a role in effecting
+behavior in client/server mode.
+
+Settings/Preferences File
+"""""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/config`` 
+* Purpose: Holds user settings from :ref:`preferences window <Preferences Window>`
+  plus numerous other settings such as default attributes for operators and plots,
+  default database read options, default color tables, as well as the
+  enabled/disabled state of various plot, operator and database plugins.
+* Written: when user :ref:`saves settings <How to Save Settings>`.
```

Comment:
You are inconsitently capitalizing the first word after the colon in this and subsequent sections. They should be made consistent.